docs/plugins/integrations.asciidoc

```diff
@@ -0,0 +1,228 @@
+
+[float]
+[[cms-integrations]]
+=== CMS integrations
+
+[float]
+==== Supported by the community:
+
+* http://drupal.org/project/search_api_elasticsearch[Drupal]:
+  Drupal Elasticsearch integration via Search API.
+
+* https://drupal.org/project/elasticsearch_connector[Drupal]:
+  Drupal Elasticsearch integration.
+
+* http://searchbox-io.github.com/wp-elasticsearch/[Wp-Elasticsearch]:
+  Elasticsearch WordPress Plugin
+
+* https://github.com/wallmanderco/elasticsearch-indexer[Elasticsearch Indexer]:
+  Elasticsearch WordPress Plugin
+
+* https://doc.tiki.org/Elasticsearch[Tiki Wiki CMS Groupware]:
+  Tiki has native support for Elasticsearch. This provides faster & better
+  search (facets, etc), along with some Natural Language Processing features
+  (ex.: More like this)
+
+[float]
+[[data-integrations]]
+=== Data import/export and validation
+
+NOTE: Rivers were used to import data from external systems into Elasticsearch prior to the 2.0 release. Elasticsearch 
+releases 2.0 and later do not support rivers.
+
+[float]
+==== Supported by the community:
+
+* https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html[Logstash output to Elasticsearch]:
+  The Logstash `elasticsearch` output plugin.
+* https://www.elastic.co/guide/en/logstash/current/plugins-inputs-elasticsearch.html[Elasticsearch input to Logstash]
+  The Logstash `elasticsearch` input plugin.
+* https://www.elastic.co/guide/en/logstash/current/plugins-filters-elasticsearch.html[Elasticsearch event filtering in Logstash]
+  The Logstash `elasticearch` filter plugin.
+* https://www.elastic.co/guide/en/logstash/current/plugins-codecs-es_bulk.html[Elasticsearch bulk codec]
+  The Logstash `es_bulk` plugin decodes the Elasticsearch bulk format into individual events. 
+
+[float]
+==== Supported by the community:
+
+* https://github.com/jprante/elasticsearch-jdbc[JDBC importer]:
+  The Java Database Connection (JDBC) importer allows to fetch data from JDBC sources for indexing into Elasticsearch (by Jörg Prante)
+
+* https://github.com/reachkrishnaraj/kafka-elasticsearch-standalone-consumer[Kafka Standalone Consumer]:
+  Easily Scalable & Extendable Kafka Standalone Consumer that reads messages from Kafka, then processes and indexes the messages in ElasticSearch
+
+* https://github.com/ozlerhakan/mongolastic[Mongolastic]:
+  A tool that clones data from ElasticSearch to MongoDB and vice versa
+
+* https://github.com/Aconex/scrutineer[Scrutineer]:
+  A high performance consistency checker to compare what you've indexed
+  with your source of truth content
+
+[float]
+[[deployment]]
+=== Deployment
+
+[float]
+==== Supported by Elasticsearch:
+
+* https://github.com/elasticsearch/puppet-elasticsearch[Puppet]:
+  Elasticsearch puppet module.
+
+[float]
+==== Supported by the community:
+
+* http://github.com/elasticsearch/cookbook-elasticsearch[Chef]:
+  Chef cookbook for Elasticsearch
+
+This project appears to have been abandoned:
```

Comment:
I'm not sure it's up to us to decide/canonize whether a project has been abandoned in our docs?